package proto_ff;

import "yd_fieldoptions.proto";


message equipequipattributeDesc
{
	optional int32 valueMin = 1[(yd_fieldoptions.field_cname) = "最小数值"];
	optional int32 type = 2[(yd_fieldoptions.field_cname) = ""];
	optional int32 valueMax = 3[(yd_fieldoptions.field_cname) = "最大数值"];
}

message equipequip
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "装备ID"];
	optional string name = 2[(yd_fieldoptions.field_cname) = "装备名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 wearQuality = 3[(yd_fieldoptions.field_cname) = "装备品阶"];
	optional int32 position = 4[(yd_fieldoptions.field_cname) = "装备位置"];
	optional string profession = 5[(yd_fieldoptions.field_cname) = "职业限制", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 professionLv = 6[(yd_fieldoptions.field_cname) = "转职等级"];
	optional int32 level = 7[(yd_fieldoptions.field_cname) = "穿戴等级限制"];
	optional int32 quality = 8[(yd_fieldoptions.field_cname) = "装备品质"];
	optional int32 star = 9[(yd_fieldoptions.field_cname) = "装备星级"];
	optional int32 refineAttributeDown = 10[(yd_fieldoptions.field_cname) = "星级属性数量下限"];
	optional int32 refineAttributeUp = 11[(yd_fieldoptions.field_cname) = "星级属性数量上限"];
	optional string refineAttribute_typeLibrary = 12[(yd_fieldoptions.field_cname) = "随机星级属性库", (yd_fieldoptions.field_bufsize) = 60];
	optional string refineAttribute_valueMin = 13[(yd_fieldoptions.field_cname) = "星级属性最小值", (yd_fieldoptions.field_bufsize) = 60];
	optional string refineAttribute_valueMax = 14[(yd_fieldoptions.field_cname) = "星级属性最大值", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 blueStarNum = 15[(yd_fieldoptions.field_cname) = "蓝色星级属性条数"];
	optional string blueStar_type = 16[(yd_fieldoptions.field_cname) = "蓝色星级属性类型库", (yd_fieldoptions.field_bufsize) = 60];
	optional string blueStar_lv = 17[(yd_fieldoptions.field_cname) = "蓝色星级等级间隔差", (yd_fieldoptions.field_bufsize) = 60];
	optional string blueStar_number = 18[(yd_fieldoptions.field_cname) = "蓝色星级每级增加值", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 isCanbind = 19[(yd_fieldoptions.field_cname) = "是否绑定"];
	optional int32 isTradeBind = 20[(yd_fieldoptions.field_cname) = "是否交易绑定"];
	optional int32 sellPrice = 21[(yd_fieldoptions.field_cname) = "出售价格"];
	optional string icon = 22[(yd_fieldoptions.field_cname) = "装备图标名", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 time = 23[(yd_fieldoptions.field_cname) = "限时装备"];
	optional int32 Combat = 24[(yd_fieldoptions.field_cname) = "战斗力（不含蓝星属性）"];
	optional int32 broadcast = 25[(yd_fieldoptions.field_cname) = "是否播放跑马灯"];
	optional string meltingResult = 26[(yd_fieldoptions.field_cname) = "熔炼产出", (yd_fieldoptions.field_bufsize) = 60];
	repeated equipequipattributeDesc attribute = 27[(yd_fieldoptions.field_cname) = "基础属性", (yd_fieldoptions.field_arysize) = 2];
}


message Sheet_equipequip
{
	repeated equipequip equipequip_List = 1[(yd_fieldoptions.field_arysize)=4000];
}
